components/prism-birb.js

```diff
@@ -0,0 +1,23 @@
+Prism.languages.birb = Prism.languages.extend('clike', {
+	'class-name': [
+		/\b[A-Z](?:[\d_]*[a-zA-Z]\w*)?\b/,
+
+		// matches variable and function return types (parameters as well).
+		/\b[A-Z]\w*(?=\s+\w+\s*[;,=()])/
+	],
+	'string': {
+		pattern: /r?("|')(?:\\.|(?!\1)[^\\])*\1/,
```

Comment:
What is the `r?` for?

Reply:
I am adding "raw" strings to Birb sometime today, any string starting with `r` as in ```
r`foo\n`
``` would be a raw string (treats \ as a literal and not an escape character)